src/pages/documentation/adobe-analytics-mobile-services/index.md

```diff
@@ -105,33 +105,17 @@ The Mobile Services extension depends on the Core extension, which includes the
 
 iOS AEP (3.x) Mobile Services extension only supports Cocoapods. It does **not** work with Swift Package Manager or when included in your app as an XCFramework dependency.
```

Comment:
It will be difficult for us to maintain the docs above in both locations, so I recommend removing this content from the ACP repo and keep only the sample code. This applies for all index.md files. 
You can keep the steps starting with "Add the X extension to your app" and add a short intro on top saying that for configuration settings go to latest docs (can we link to main aep docs from here?).
Feel free to take this one in a separate PR, seems big